exec/cnex/lib/file_win32.c

```diff
@@ -0,0 +1,243 @@
+#include <io.h>
+#include <windows.h>
+
+#include "cell.h"
+#include "enums.h"
+#include "exec.h"
+#include "nstring.h"
+#include "number.h"
+#include "stack.h"
+#include "util.h"
+
+
+static void handle_error(TExecutor *exec, DWORD error, TString *path)
+{
+    switch (error) {
+        case ERROR_ALREADY_EXISTS:      exec->rtl_raise(exec, "FileException.DirectoryExists", TCSTR(path), number_from_uint32(error));     break;
+        case ERROR_ACCESS_DENIED:       exec->rtl_raise(exec, "FileException.PermissionDenied", TCSTR(path), number_from_uint32(error));    break;
+        case ERROR_PATH_NOT_FOUND:      exec->rtl_raise(exec, "FileException.PathNotFound", TCSTR(path), number_from_uint32(error));        break;
+        case ERROR_FILE_EXISTS:         exec->rtl_raise(exec, "FileException.Exists", TCSTR(path), number_from_uint32(error));              break;
+        case ERROR_PRIVILEGE_NOT_HELD:  exec->rtl_raise(exec, "FileException.PermissionDenied", TCSTR(path), number_from_uint32(error));    break;
+        default:
+        {
+            char err[1024];
+            snprintf(err, 1024, "%s: %d", TCSTR(path), error);
```

Comment:
Same comment about using a constant and not repeating the size value.

Reply:
Used MAX_PATH for size of err.